Sources/PackageGraph/DependencyResolver.swift

```diff
@@ -1201,13 +1201,13 @@ public class DependencyResolver {
     public var containers: [PackageReference: Container] {
         return fetchCondition.whileLocked({
             _fetchedContainers.spm_flatMapValues({
-                try? $0.dematerialize()
+                try? $0.get()
             })
         })
     }
 
     /// The list of fetched containers.
-    private var _fetchedContainers: [PackageReference: Basic.Result<Container, AnyError>] = [:]
+    private var _fetchedContainers: [PackageReference: Swift.Result<Container, AnyError>] = [:]
```

Comment:
We can drop `Swift.` here, right?

Reply:
Unfortunately, `DependencyResolver.swift` defines another enum named `Result` which represents similar concept, but with 3 cases. `Swift.` needs to stay to not confuse the compiler.

Reply:
ah ok, makes sense.

Reply:
I wonder if we should eventually change the name of that enum, if use of the stdlib Result becomes ubiquitous.  I know it's not technically a name conflict, but could still lead to confusion for anyone reading the code.

Reply:
I agree. The concept mismatch in this case can be confusing. I can address this in another PR, do you have any name suggestions that could represent `DependencyResolver.Result` better?